pkgs/development/python-modules/spylls/default.nix

```diff
@@ -0,0 +1,67 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+
+, pythonOlder
+
+, pythonRelaxDepsHook
+
+, poetry-core
+
+, pytest
+}:
+
+buildPythonPackage rec {
+  pname = "spylls";
+  version = "0.1.7";
+  pyproject = true;
+
+  disabled = pythonOlder "3.7";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-cEWJLcvTJNNoX2nFp2AGPnj7g5kTckzhgHfPCgyT8iA=";
+  };
+
+  postPatch = ''
+    substituteInPlace pyproject.toml \
+    --replace poetry.masonry.api poetry.core.masonry.api \
+    --replace "poetry>=" "poetry-core>="
```

Comment:
Could you suggest the use of poetry-core to the upstream?
https://github.com/python-poetry/poetry-core?tab=readme-ov-file#why-is-this-required

Reply:
Done
https://github.com/zverok/spylls/pull/27

Reply:
@natsukium